lib/pod/command/plugins/installed.rb

```diff
@@ -41,25 +42,46 @@ def print_compact_list(plugins)
           max_length = plugins.map { |p| p.name.length }.max
           plugins.each do |plugin|
             name_just = plugin.name.ljust(max_length)
-            UI.puts_indented " - #{name_just} : #{plugin.version}"
+            hooks = registered_hooks(plugin)
+            hooks_list = hooks.empty? ? '' : " (#{hooks.join(' & ')} hook)"
```

Comment:
It's part of active support which CocoaPods explicitly depends upon and require's it.